articles/personal_growth.md

```diff
@@ -0,0 +1,33 @@
+---
+authorName: Sabrina Lange
+authorGithubUsername: sweinzie 
+issue: 97
+title: Personal Growth - Why every Project Should Have Gardeners
+---
+# {{page.title}}
+
+Personal Growth - Why every Project Should Have Gardeners
+
+Everyone started small. Even the tallest and strongest tree started as a seed and over time, thanks to a lot of sunlight, rainfall and good soil it became a deep-rooted, reliable and sturdy tree. And every year in the summertime it offers shade and fruits to anyone who appreciates it.
+Very much the same also applies to us, people who are involved in project work, where we try to give our best effort to make the project a success. 
+
+We work in projects as a subject matter expert, as an experienced project manager or as a newcomer that will need support from the team to grow to a new leader. Roles and tasks are assigned based on experiences we made in previous assignments. Experiences from former projects are our soil. We are able to grow on new and demanding tasks but will fail on overwhelming tasks. 
+If our responsibility lies within our comfort zone and is business as usual, we will not reach our limits. We need challenges such as finding new solutions and new approaches to grow. Just as a plant that needs more and more space to become bigger and greater we need space and the opportunity to make our own decisions and take responsibility.
+We need managers who see the potential of the project but also required skills for staffing such a project with team members. Managers that are aware of the strengths and weaknesses of each person and if there is a realistic feasibility that person can perform well and do a great job within the project.
+
+Besides the soil a plant needs right at the beginning also some kind of pillar. As long as the roots are not deeply anchored, and the stem is still fragile it is important that the plant is supported. It is exactly like that with us trying to grow in our projects. By having all these challenges and opportunities to grow, sometimes we will make a mistake. Sometimes we are not fulfilling expectation of our stakeholders, maybe our estimation or planning was wrong due to wrong assumptions, we took wrong decisions or communicated not well or the way we acted was not as good as expected from our sponsor. These are the moments we need support and orientation when this situation takes us to our limits. We need someone taking care of us, helping that we will not break in the storm but become stronger and are prepared for the next time we face a similar situation.
```

Comment:
Sprache: "It is exactly like that with us trying to grow in our projects." -> tönt etwas komisch

Kürzungspotential: "Sometimes we are not fulfilling expectation of our stakeholders, maybe our estimation or planning was wrong due to wrong assumptions, we took wrong decisions or communicated not well or the way we acted was not as good as expected from our sponsor." -> ganz weglassen (es ist ja eigentlich klar, was ein Mistake ist) oder vielleicht so: "Sometimes we are not fulfilling expectation of our stakeholders, not communicating well, not taking right decisions, not acting as expected."